group_vars/matrix_servers

```diff
@@ -488,6 +488,28 @@ matrix_sms_bridge_homeserver_token: "{{ matrix_synapse_macaroon_secret_key | pas
 #
 ######################################################################
 
+######################################################################
+#
+# matrix-bridge-heisenbridge
+#
+######################################################################
+
+# We don't enable bridges by default.
+matrix_heisenbridge_enabled: false
+
+matrix_heisenbridge_systemd_wanted_services_list: |
+  {{
+    (['matrix-synapse.service'] if matrix_synapse_enabled else [])
+    +
+    (['matrix-nginx-proxy.service'] if matrix_nginx_proxy_enabled else [])
+  }}
+
```

Comment:
`matrix_heisenbridge_appservice_token` and `matrix_heisenbridge_homeserver_token` (discussed in other review comments) will need to go here.

You can take inspiration from another bridge: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/2606b41b166cc8726d5fd096fbe395ee01ff9cf4/group_vars/matrix_servers#L239-L241

Make sure to change `'fb.as.token'` and `'fb.hs.token'` to something unique (`'heisen.as.tok'` and `'heisen.hs.tok'` would work well). These strings need to be less than 16 characters long to avoid Ansible errors on certain platforms.